examples/src/main/scala/org/apache/spark/examples/mllib/MovieLensALS.scala

```diff
@@ -74,6 +75,9 @@ object MovieLensALS {
       opt[Unit]("implicitPrefs")
         .text("use implicit preference")
         .action((_, c) => c.copy(implicitPrefs = true))
+      opt[Unit]("validateRecommendation")
```

Comment:
We can switch the evaluation metric between RMSE and MAP via a `--metrics` option. This is more specific than `validateRecommendation`.